Misc/NEWS.d/next/Library/2022-11-23-23-58-45.gh-issue-94943.Oog0Zo.rst

```diff
@@ -0,0 +1 @@
+When inheriting from a dataclass, only show the field names in the value section of the member repr(), and not the dataclass' class name.
```

Comment:
```suggestion
Add :ref:`enum-dataclass-support` to the
:class:`~enum.Enum` :meth:`~enum.Enum.__repr__`.
When inheriting from a :class:`~dataclasses.dataclass`,
only show the field names in the value section of the member :func:`repr`,
and not the dataclass' class name.
```

* Link the docs section with more information
* Add appropriate Sphinx cross references and syntax
* Ensure <80 characters per line

Reply:
I thought blurb would take care of the 80 character line.  Huh.